src/io/flutter/FlutterBundle.properties

```diff
@@ -123,15 +123,30 @@ flutter.module.create.settings.help.project_type.label=Project type:
 flutter.module.create.settings.help.project_type.description.app=Select an "Application" when building for end users.
 flutter.module.create.settings.help.project_type.description.plugin=Select a "Plugin" when exposing an Android or iOS API for developers.
 flutter.module.create.settings.help.project_type.description.package=Select a "Package" when creating a pure Dart component, like a new Widget.
+flutter.module.create.settings.help.type.application=An "application" is a stand-alone app that can be installed and run on a mobile device.
 flutter.module.create.settings.help.type.package=A "package" is a pure-Dart component, like a new Widget, with no native code.
 flutter.module.create.settings.help.type.plugin=A "plugin" is used to expose an Android or iOS API for developers.
 
+module.wizard.language.name_kotlin=Use Kotlin for Android code (defaults to Java)
```

Comment:
I might suggest "Use the Kotlin language for Android. (defaults to the Java language)"

Reply:
Using "Android/iOS code" links the button to "platform-specific code" in the description at the top of the page.

Reply:
Actually, I'd suggest changing the type of control, I think this would be easier to understand with regular radio buttons:

```
Android platform: (*) Java          ( ) Kotlin
iOS platform:     (*) ObjectiveC    ( ) Swift
```

Reply:
Android Studio uses a check box to determine language. Wouldn't it be better to follow the same style as the host IDE?